buildpacks/jvm/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
```

Comment:
(optional) Technically this is really two sets of changes to the buildpack, right? An `## Added` for adding support for new versions, and `## Changed` for changing the default versions when using a major version alias or no version has been specified at all. Currently the changelog only explicitly mentions the latter, and the former has to be inferred.